src/test/java/cucumber/steps/JQueryStepDefinitions.java

```diff
@@ -0,0 +1,39 @@
+package cucumber.steps;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+import pages.jQueryUIPages.JQueryUAHomePage;
+import pages.jQueryUIPages.SearchResultPage;
+import ru.yandex.qatools.allure.annotations.Step;
+import service.DriverSingleton;
+
+public class JQueryStepDefinitions {
+    @Step("Step 1: Opening JQuery site")
```

Comment:
Bad practice to mix Definitions and Steps in one class.
@Step should be placed strictly before method, you used.
Main idea about cucumber + Allure --> we should see Given/When/Then strings, not @Step.
@Step is useful for testng tests.

Reply:
removed steps